src/configs/all.ts

```diff
@@ -132,6 +132,7 @@ export const rules = {
     "max-file-line-count": [true, 1000],
     "max-line-length": [true, 120],
     "no-default-export": true,
+    "no-irregular-whitespaces": true,
```

Comment:
the eslint rule is called [`no-irregular-whitespace`](http://eslint.org/docs/rules/no-irregular-whitespace). note the lack of trailing "s". please name it exactly the same (it's potentially confusing otherwise).